walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,76 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import java.util.List;
 
 public final class FindMeetingQuery {
+
+  /**
+   * A comparator for sorting events by their start time in ascending order.
+   */
+  private static Comparator<Event> eventsComparator = new Comparator<Event>() {
+    @Override
+    public int compare(Event a, Event b) {
+      return TimeRange.ORDER_BY_START.compare(a.getWhen(), b.getWhen());
+    }
+
+  };
```

Comment:
fyi: There's a utility function that makes this easier.
```suggestion
  private static Comparator<Event> eventsComparator = 
          Comparator.comparing(Event::getWhen, TimeRange.ORDER_BY_START);
```